ts/screens/wallet/payment/components/TransactionSummary.tsx

```diff
@@ -166,10 +166,20 @@ type Props = Readonly<{
 export const TransactionSummary = (props: Props): React.ReactElement => {
   const isLoading = pot.isLoading(props.paymentVerification);
 
+  const getRecepientName = (recipient: EnteBeneficiario) => {
```

Comment:
What do you think about move this in utils/strings file? Could it be used by other screens?